### DISCUSSION TITLE
UR ROS2 Driver - Windows 10 Support

### DISCUSSION THREAD

**Discussion by cwl5535** [UniversalRobots/Universal_Robots_ROS2_Driver#375] [Q&A]
Hello! 

I saw on that the UR ROS2 Driver has been officially released and was wondering if this new release included support for Windows 10? I know previous releases of the ROS1 driver only had native support for Ubuntu. 

Thank you!
Colton

**Comment by fmauch** [comment]
There is currently no native windows support. I know that people have been using it using WSL2 as [here](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/issues/252).

**Reply by cwl5535** [reply]
Thank you! Do you know if there are any plans to have native support for Windows as ROS2 begins adding support for Windows?

**Reply by fmauch** [reply]
This is currently not on our top priority list.
